packages/api/src/db/block.ts

```diff
@@ -102,8 +102,15 @@ export interface ListicleBlock {
 export interface LinkPageBreakBlock {
   type: BlockType.LinkPageBreak
   text: string
+  richText: RichTextNode[]
   linkURL: string
   linkText: string
+  linkTarget: string
+  hideButton: boolean
```

Comment:
I don't see the point of this boolean. Can't you just make `linkText` optional and if it's empty don't display a button?

Reply:
its an options, which makes the whole element clickable, it keeps the value of the linkText label to reactivate, to make it more understandable and flexible it better to have a toggler with diescription

Reply:
How does that work with links inside the richText editor?

Reply:
links inside editor are clear text links, this is a button and if you select an image left or right its clickable.

If you use RTE only i assume you cant click the link now (but think the problem you try to find is solvable by css)